tests/test_views.py

```diff
@@ -1015,6 +1003,17 @@ def test_get_course_enrollment_page(
             configuration_helpers_mock,
             render_to_response_mock,    # pylint: disable=unused-argument
     ):
+        dummy_price_details_mock = mock.MagicMock()
+        dummy_price_details_mock.return_value = {
+            'code': 'mockcode',
+            'price': 100,
+            'final_price': 50,
+        }
+        price_details_mock = mock.MagicMock()
+        method_name = 'enterprise.user.{}/products/sku-professional/price.get'.format(self.user.username)
+        attrs = {method_name: dummy_price_details_mock}
+        price_details_mock.configure_mock(**attrs)
+        ecommerce_api_client_mock.return_value = price_details_mock
```

Comment:
@replaceafill As mentioned on OC-2660, if this code can no longer live in `setUp`, could you please move it to a separate method to reduce the amount of duplication in the test code?